runtime/doc/news.txt

```diff
@@ -112,6 +112,11 @@ The following changes may require adaptations in user config or plugins.
 • 'termguicolors' is enabled by default when Nvim is able to determine that
   the host terminal emulator supports 24-bit color.
 
+• Treesitter highlight groups have been renamed to be more in line with
```

Comment:
Super nit and silly, but the lack of a naming convention is starting to annoy me: Is it Treesitter, treesitter, Tree-sitter, 3-sitter, ninja-gardener??

To be serious here, using both "Treesitter" and "tree-sitter" in the same documentation sentence looks quite confusing to me.

Reply:
It should be "Tree-sitter" in prose (that is how they [describe themselves](https://tree-sitter.github.io/tree-sitter/)) and `treesitter` in code.

Reply:
Justin disagrees: it's the "tree-sitter" project, but "treesitter" in _our_ code. This is how we enforce commit messages, for example. If there's a `tree-sitter`, _that_ one needs to be changed.

Reply:
Let's continue the discussion in #27099, as it's a bit off-topic for this PR.

Reply:
> the lack of a naming convention is starting to annoy me: Is it Treesitter, treesitter, Tree-sitter, 3-sitter, ninja-gardener??

That's exactly why I suggest "treesitter". Multi-word names are horrible for this reason, it affects all kinds of names including filenames and symbols.